scripts/install_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+curl -sSL https://get.docker.com/ubuntu/ | sh
```

Comment:
Forget it, just realized I'm on a 32 bit ubuntu and I see you also modified the vagrantfile to use 64 bit. never mind, this LGTM.